streaming-binlog/src/test/java/org/openmrs/analytics/FhirConverterTest.java

```diff
@@ -92,6 +107,24 @@ public void shouldFetchFhirResourceAndUploadToFhirStore() throws IOException {
 		Mockito.verify(parquetUtil, Mockito.never()).write(Mockito.<Resource> any());
 	}
 	
+	@Test
+	public void shouldGetUuidFromParent() throws SQLException, PropertyVetoException, ClassNotFoundException {
+		
+		Map<String, String> messageBody = DebeziumTestUtil.genExpectedBody();
+		Map<String, Object> messageHeaders = DebeziumTestUtil.genExpectedHeaders(Operation.CREATE, "patient");
+		String uuid = getUuidUtil.getUuid(Mockito.anyString(), Mockito.anyString(), Mockito.anyString());
+		
+		lenient().when(jdbcConnectionUtil.createStatement()).thenReturn(statement);
+		
+		lenient().when(getUuidUtil.getUuid(Mockito.anyString(), Mockito.anyString(), Mockito.anyString())).thenReturn(uuid);
+		
+		// Actual event that will trigger process().
+		eventsProducer.sendBodyAndHeaders(messageBody, messageHeaders);
+		
+		Mockito.verify(getUuidUtil, times(1)).getUuid(Mockito.anyString(), Mockito.anyString(), Mockito.anyString());
+		Mockito.verify(getUuidUtil).getUuid(Mockito.anyString(), Mockito.anyString(), Mockito.anyString());
```

Comment:
I think it is better to also verify that `openmrsUtil.fetchFhirResource` has been called with the right arguments, i.e., with the `uuid` above.